### PR TITLE
Fixes cult communication not working while buckled or cuffed

### LINKED ISSUE #73701: Cult actions that should be usable cannot be used while sitting on a chair.
## Reproduction:

~~I suspect this is a result of #73513~~ Nevermind its not
I've not had a chance to try all spells but I know this impacts all cult spells. 
Become a cultist, buckle yourself to a chair, be unable to commune.

### PR DESCRIPTION
**Pull request by Lufferly**
## About The Pull Request

Fixes #73701
Lets cult communion and assert leadership (lol) be usable while your hands are blocked or you are immobile.

## Why It's Good For The Game

Fixes a bug, and these are supposed to be whispers or whatever so it does not make much sense that being cuffed would stop that. Plus the ability makes you whisper, so its not without risks if you do call for help while cuffed in brig.
Also assert leadership makes sense I guess?

## Changelog

:cl: Seven
fix: Cult communion and assert leadership can be used while your hands are blocked or you are immobile
/:cl:

### CODE REVIEW

File: code/modules/antagonists/cult/cult_comms.dm [hunk 1]
```diff
@@ -7,8 +7,21 @@
...
 	ranged_mousepointer = 'icons/effects/mouse_pointers/cult_target.dmi'
 
+/datum/action/innate/cult/New()
+	..()
+	// If we dont need hands, remove the check from check_flags
+	if(does_not_need_hands == TRUE)
+		check_flags = check_flags & ~AB_CHECK_HANDS_BLOCKED
+	// If we dont need to be mobile, remove the check from check_flags
+	if(does_not_need_mobility == TRUE)
+		check_flags = check_flags & ~AB_CHECK_IMMOBILE
```
Comment (Ghommie) [inline code/modules/antagonists/cult/cult_comms.dm:23]: Why didn't you just remove those check_flags in the definitions of those two actions? 😵

Comment (Lufferly) [reply code/modules/antagonists/cult/cult_comms.dm:23]: I dunno, do you want me to do that instead?

Comment (Ghommie) [reply code/modules/antagonists/cult/cult_comms.dm:23]: That'd be better, yes.

Comment (Lufferly) [reply code/modules/antagonists/cult/cult_comms.dm]: done, thanks

**Comment by BlueMemesauce** [comment]
This is a big balance change, but it will take a bit for me to explain why.
So cultists have a rune called "Summoning Rune", which can instantly teleport any cultist to it. However, the rune doesn't work if the cultist is buckled or being dragged. So when you want to deconvert a cultist, you have to make sure they're buckled so they can't escape, since it takes a while for the holy water to kick in.

The thing is, unless you're constantly being stunned or being moved around, you can always eventually escape being buckled. Normally, you'd be quickly caught and buckled back down again, giving a very small window of opportunity. 

The rune is currently seldom used due to this as you need someone to know that you've been caught and constantly spamming summon cultist until they can summon you. But with being cultists able to communicate while restrained, summon cultist becomes much easier to coordinate. You don't even need someone to know you've been caught, you can just say, "hey guys I've been caught prep the summon rune". And then just tell them you've broken free of the buckle and you'll be teleported away before sec even has a chance to realize that you've become unbuckled.

TL;DR: This change will make deconversion much more difficult

A possible fix for this would be making Holy Water disable the use of the ability. But there would still be problems if security doesn't have holy water ready to go and they're waiting on the chaplain to come by. They'd have to sit there constantly stunning the cultist until deconversion.

File: code/modules/antagonists/cult/cult_comms.dm [hunk 2]
```diff
@@ -90,6 +92,9 @@
 /datum/action/innate/cult/mastervote
 	name = "Assert Leadership"
 	button_icon_state = "cultvote"
+	// So you can use it while your hands are cuffed or you are bucked
+	// If you want to assert your leadership while handcuffed to a chair, be my guest
+	check_flags = AB_CHECK_INCAPACITATED|AB_CHECK_CONSCIOUS
```
Comment (MrMelbert) [inline code/modules/antagonists/cult/cult_comms.dm:97]: Leave `AB_CHECK_HANDS_BLOCKED` for this, the actual mastervote action blacklists handcuffed people from being voted up in the process

**Review by MrMelbert — CHANGES_REQUESTED** [review]
Leaving balance concerns aside

**Comment by nukularpower** [comment]
It seems fairly silly to me that this is an issue to begin with - there are many "spells" (meaning various abilities) that should certainly be usable in crit, buckled, etc, but are not on TG for some reason.  In fact, as I recall, there was a prior PR that was supposed to fix it, but didnt seem to work, or maybe I just dont understand what it meant, though I cant seem to find it now.

**Comment by san7890** [comment]
yeah ultimately it's silly to not be able to speak when your hands are restrained
